### ISSUE TITLE
Does not work w/ Ember 2.0

### ISSUE THREAD

**Issue by Randuin** [Semantic-Org/Semantic-UI-Ember#49]
```
master⚡ ⇒ ember serve --proxy http://localhost:3000

version: 1.13.8
Proxying to http://localhost:3000
Livereload server on http://localhost:49153
Serving on http://localhost:4200/
File: templates/components/ui-dropdown.hbs
Assertion Failed: Using `{{view}}` or any path based on it ('templates/components/ui-dropdown.hbs' @ L1:C18) has been removed in Ember 2.0
Error: Assertion Failed: Using `{{view}}` or any path based on it ('templates/components/ui-dropdown.hbs' @ L1:C18) has been removed in Ember 2.0
```

**Comment by MartinMalinda** [comment]
Are you using the prep-1.0 branch?

**Comment by cklimkowsky** [comment]
I'm getting similar errors as well:
```
DEPRECATION: Using `{{view}}` or any path based on it ('templates/components/ui-dropdown.hbs' @ L18:C0) has been deprecated.
```

How do you use the prep-1.0 branch?

**Comment by MartinMalinda** [comment]
Install it with npm instead
```npm install --save git://github.com/Semantic-Org/Semantic-UI-Ember.git#prep-1.0```

**Comment by nathanpalmer** [comment]
We just released version 0.9.0 that works with 2.0 (which has some breaking changes in it.) Let me know if you're still having issues there.

**Comment by Nininea** [comment]
I run this command :
ember install https://github.com/Semantic-Org/Semantic-UI-Ember.git#prep-0.9.0 but ends with errors, can you help me?
